File: beginner/java/2717.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int n, a, b;

        n = scanner.nextInt();
        a = scanner.nextInt();
        b = scanner.nextInt();

        if((a + b) > n)
            System.out.println("Deixa para amanha!");
        else
            System.out.println("Farei hoje!");

        scanner.close();
    }
}

File: beginner/javascript/2717.js
const fs = require('fs');
const input = fs.readFileSync('stdin', 'utf8');
const lines = input.trim().split('\n');

var n, a, b;

n = parseInt(lines.shift());

var valores = lines.shift().split(" ");

a = parseInt(valores[0]);
b = parseInt(valores[1]);

if((a + b) > n)
    console.log("Deixa para amanha!");
else
    console.log("Farei hoje!");